docs/Configuration.md

```diff
@@ -32,6 +32,60 @@ SQLite            | `SQLite`
 
 TBD
 
+## LANGUAGE section
+
+The LANGUAGE section has one key, `locale`.
+
+The value of the `locale` key is a space separated list of
+`locale tags` where each tag must match the regular expression
+`/^[a-z]{2}_[A-Z]{2}$/`.
+
+The two first characters of a `locale tag` are intended to be an
+[ISO 639-1] two-character language code and the two last characters
+are intended to be an [ISO 3166-1 alpha-2] two-character country code.
+A `locale tag` is a locale setting for the available translation
+of messages without ".UTF-8", which is implied.
+
+Adding a new `locale tag` to the configuration requires that the
```

Comment:
What kind of requirement is this? Is it a requirement to avoid unspecified behavior? It should be clarified what happens if the requirement is not met.

Maybe the "Each locale set" paragraph is the clarification? In that case that paragraph should be integrated here.

Reply:
I think it was fine, but I reworded it.